code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -307,6 +307,16 @@
 		var/obj/pill = AP.target
 		if(pill)
 			pill.forceMove(src)
+	for(var/datum/action/item_action/hands_free/activate_signaler/AS in owner.actions)
+		AS.Remove(owner)
+		var/obj/sig = AS.target
+		if(sig)
+			sig.forceMove(src)
```

Comment:
```suggestion
		sig?.forceMove(src)
```

Reply:
That ? is a tricky little bugger. Mostly seen it in like "is x true? if so 1 else 2" in the form of "var? : 1 || 2" or however it's done.